typings/vscode-tasks.d.ts

```diff
@@ -32,25 +32,21 @@ declare module "vscode-tasks" {
     export interface BaseTaskConfiguration {
```

Comment:
Why do we have a copy of these typings? In the 1.1.6 version of the vscode package I see that these types are already defined in vscode.d.ts.

Reply:
Not sure why we originally had a copy of this. @DustinCampbell may be able to answer that question. However, the current typings do not have the old Tasks schema, specifically isBuildCommand.

Reply:
Back in the day, it wasn't included in vscode.d.ts.